src/maintainer/adding_pkgs.rst

```diff
@@ -3,15 +3,19 @@
 Contributing packages
 *********************
 
+Process
+===================
 
-To submit a package to the ``conda-forge`` channel, add its ``recipe`` and licence to the ``staged-recipes`` repository and create a pull request. Once the pull request is merged, the package becomes available
-on the ``conda-forge`` channel. Note that contributing a package makes you the ``maintainer`` of that package.
+To submit a package to conda-forge one has to follow the steps:
 
-A maintainer is responsible for maintaining the feedstock repository and packages as well as their future versions and has push access to the feedstock repositories of only the packages it maintains. 
-You can learn more about the roles of a maintainer `here. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__
+* Step 1. The staging process (add recipe and license)
+ To submit a package to the ``conda-forge`` channel, one has to add its recipe and license, with the help of the `staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_, to the `staged-recipes repository <https://github.com/conda-forge/staged-recipes>`_ and create a pull request.
```

Comment:
Same second person imperative tense here as above. "To submit a package to the conda-forge channel, add its recipe and license..."